NetKAN/ToadicusToolsContinued.netkan

```diff
@@ -0,0 +1,14 @@
+{
+  "spec_version": "v1.4",
+  "install": [
+    {
+      "install_to": "GameData",
+      "find": "ToadicusTools"
+    }
+  ],
+  "$vref": "#/ckan/ksp-avc",
+  "$kref": "#/ckan/spacedock/1069",
+  "x_via": "Automated Linuxgurugamer CKAN script",
+  "identifier": "ToadicusToolsContinued",
+  "license": "unknown"
```

Comment:
This is the line in question, it says unknown while the old version of ToadicusTools said BSD-3-clause